modules/enabling-scanner-v4-after-helm-installation.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * operating/examine-images-for-vulnerabilities.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="enabling-scanner-v4-after-helm-installation_{context}"]
+= Enabling RHACS Scanner V4 after Helm installation
+
+[role="_abstract"]
+Scanner V4 is not enabled by default, but you can enable Scanner V4 after installation.
+
+If you want to enable {product-title-short} on secured clusters by using Helm charts, you must first enable Scanner V4 on the cluster where Central is installed.
+
+.Procedure
+
+. Create a `values-public.yaml` file that contains the following lines:
++
+[source,yaml]
+----
+scannerV4:
+    disable: false
+----
+. Run the command to update {product-title-short} with customizations as described in "Changing configuration options after deploying the central-services Helm chart":
++
+[source,terminal]
+----
+helm upgrade -n stackrox \
+  stackrox-central-services rhacs/central-services \
+  --reuse-values \
+  -f <path_to_init_bundle_file \
+  -f <path_to_values_public.yaml> \
+  -f <path_to_values_private.yaml>
+----
+
+After you enable Scanner V4 on the cluster where Central is installed and configure your init bundle or cluster registration secret (CRS) to allow communication between Central and the secured cluster, you must enable Scanner V4 on the secured cluster.
+
+.Procedure
+. Create a `values-public.yaml` file that contains the following lines:
++
+[source,yaml]
+----
+scannerV4:
+    scannerComponent: AutoSense
+----
```

Comment:
Mentioned in other comment, this appears to be yaml for the operator, believe this should be:

```suggestion
[source,yaml]
----
scannerV4:
    disable: false
----
```

However, consider for the helm instructions using the `--set` CLI argument for consistency (or updating everywhere to use values file)